src/astro/sql/operators/agnostic_load_file.py

```diff
@@ -34,15 +33,15 @@ def __init__(
         path: str,
         output_table: Table,
         file_conn_id: Optional[str] = "",
```

Comment:
It would be great if the following arguments could be contained within `File`:
* `path`
* `file_conn_id`
* `ndjson_normalize_sep`

Reply:
besides with ndjson_normalize_sep, all params are moved to the File instance.

Reply:
Why we didn't move ndjson?

Reply:
@kaxil Well, one thought was that property is not consumed by any function within File Class. The function(_populate_normalize_config) which uses it lives within the LoadFile Op file, and it should live at this location because it has knowledge about both file class and database class.